adal/constants.py

```diff
@@ -215,6 +215,14 @@ class AADConstants(object):
         'login.microsoftonline.us',
         'login.microsoftonline.de',
         ]
+    WHITELISTED_DOMAINS = [
+        #add dsts domains to whitelist

```

Comment:
Suggest to modify this comment to include the link to Domain Reference, something like this:

    # Define dSTS domains whitelist based on its Supported Environments & National Clouds list here
    # https://microsoft.sharepoint.com/teams/AzureSecurityCompliance/Security/SitePages/dSTS%20Fundamentals.aspx

Reply:
Added comments